statusHandler/statusMetricsProvider.go

```diff
@@ -255,7 +255,7 @@ func (sm *statusMetrics) RatingsMetrics() map[string]interface{} {
 	ratingsMetrics[common.MetricRatingsGeneralSignedBlocksThreshold] = sm.loadStringMetric(common.MetricRatingsGeneralSignedBlocksThreshold)
 
 	numSelectionChances := sm.loadUint64Metric(common.MetricRatingsGeneralSelectionChances + "_count")
-	selectionChances := make([]interface{}, 0)
+	selectionChances := make([]map[string]interface{}, 0)
```

Comment:
the items within the map seem only to be `uint64`. therefore, we could change the map from `map[string]interface{}` to `map[string]uint32`

Reply:
right :+1: